mongorm/DocumentMetaclass.py

```diff
@@ -50,7 +50,7 @@ def __new__( cls, name, bases, attrs ):
 			attrs['_collection'] = collection
 		
 		# find all fields and add them to our field list
-		for attrName, attrValue in attrs.items( ):
+		for attrName, attrValue in list(attrs.items( )):
```

Comment:
what's the go with leaving a space in the brackets? 😅

Reply:
That's how the rest of the codebase is styled, so until we add linting to the repo, don't really want to mix and match